Support/bone101/UI/bbui.js

```diff
@@ -108,8 +108,8 @@ var UI = (function() {
         // positions
         var BBposX = 283;
         var BBposY = 120;
-        var axisStartY = BBposY + 40;
-        var axisStartX = BBposX + 375;
+        var axisStartY = BBposY - 50;
```

Comment:
Please at least split the commit to change these positions from the change to add the graph draw.